tephra-hbase-compat-0.96/src/main/java/co/cask/tephra/hbase96/coprocessor/TransactionVisibilityFilter.java

```diff
@@ -106,6 +110,19 @@ public ReturnCode filterKeyValue(Cell cell) throws IOException {
       // passed TTL for this column, seek to next
       return ReturnCode.NEXT_COL;
     } else if (tx.isVisible(kvTimestamp)) {
+      if (deleteTracker.isFamilyDelete(cell)) {
+        deleteTracker.addFamilyDelete(cell);
+        if (clearDeletes) {
+          return ReturnCode.NEXT_COL;
```

Comment:
No, and you couldn't use it if there was.  The next column could contain a cell with a newer timestamp, which would _not_ be deleted and would have to be included.  That's why we need the DeleteTracker class to maintain the state of a family delete marker that has been seen.